webapp/schema.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS post;
+
+CREATE TABLE user (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  username TEXT UNIQUE NOT NULL,
+  password TEXT NOT NULL
+);
+
+CREATE TABLE task (
```

Comment:
It's a good practice to have a separate folder for database. you can keep the schemas separated in different files too. As projects grow, this practice becomes necessary.

Reply:
Good idea.